open_ce/inputs.py

```diff
@@ -408,7 +413,11 @@ def parse_args(parser, arg_strings=None):
     args = parser.parse_args(arg_strings)
     _create_env_config_paths(args)
 
-    _check_and_create_fips_packages(args, arg_strings)
+    #--fips and --build_ffmpeg together is not supported
```

Comment:
why?  I thought when you do `--build_ffmpeg` , we also build fips (i.e., the same as the `--fips`) option?

Reply:
when we do` --fips` we are invoking build of `openssl-env.yaml` and `openssl-env.yaml` also contains listing of fips complaint ffmpeg feedstock. 
So `build_ffmpeg` is not needed with `--fips`. The ffmpeg feedstock will be built by default when we provide `--fips`.